narwhal/primary/src/core.rs

```diff
@@ -22,22 +26,21 @@ use storage::CertificateStore;
 use store::Store;
 use sui_metrics::spawn_monitored_task;
 use tokio::{sync::watch, task::JoinHandle};
+
 use tracing::{debug, error, info, instrument, trace, warn};
 use types::{
     ensure,
     error::{DagError, DagError::StoreError, DagResult},
     metered_channel::{Receiver, Sender},
-    Certificate, Header, HeaderDigest, ReconfigureNotification, Round, RoundVoteDigestPair,
-    Timestamp, Vote,
+    Certificate, Header, HeaderDigest, LatestHeaderRequest, ReconfigureNotification, Round,
+    RoundVoteDigestPair, Timestamp, Vote,
 };
 
 #[cfg(test)]
 #[path = "tests/core_tests.rs"]
 pub mod core_tests;
 
-// TODO: enable below.
-// Rejects a header if it requires catching up the following number of rounds.
-// const MAX_HEADER_ROUND_CATCHUP_THRESHOLD: u64 = 20;
+const LATEST_HEADER_REQUEST_TIMEOUT_SECS: u64 = 15;
```

Comment:
instead of converting to duration every time and having a unit suffix in the name, make this const a Duration directly?

`const LATEST_HEADER_REQUEST_TIMEOUT: Duration = Duration::from_secs(15);`